src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -117,7 +118,8 @@
    (schema/optional-key :gzip-enable)                schema/Bool
    (schema/optional-key :access-log-config)          schema/Str
    (schema/optional-key :shutdown-timeout-seconds)   schema/Int
-   (schema/optional-key :post-config-script)         schema/Str})
+   (schema/optional-key :post-config-script)         schema/Str
+   (schema/optional-key :disallow-renegotiation)      schema/Bool})
```

Comment:
Since this was written our style guide has changed to not align values in a map, however to keep with the style of this file I think we should probably do that.